.gitignore

```diff
@@ -12,3 +12,12 @@ outputs/*
 ai_settings.yaml
 .vscode
 auto-gpt.json
+.idea
+venv
+.idea
+ai_settings.yaml
+app/.DS_Store
+.DS_Store
```

Comment:
Good that you added .DS_Store to gitignore, I still see lots of .DS_Store files already committed, please remove them.